openni2_launch/package.xml

```diff
@@ -17,8 +17,12 @@
   <run_depend>image_proc</run_depend>
   <run_depend>nodelet</run_depend>
   <run_depend>openni2_camera</run_depend>
+  <run_depend>python-pyusb-pip</run_depend>   
```

Comment:
this appears to be a pip dependency -- which would prevent this package from being released as debian package.